cpp/tests/copying/concatenate_tests.cpp

```diff
@@ -59,8 +57,20 @@ struct TypedColumnTest : public cudf::test::BaseFixture {
   {
     auto typed_data = static_cast<char*>(data.data());
     auto typed_mask = static_cast<char*>(mask.data());
-    thrust::sequence(thrust::device, typed_data, typed_data + data_size());
-    thrust::sequence(thrust::device, typed_mask, typed_mask + mask_size());
+    std::vector<char> h_data(data_size());
+    thrust::sequence(h_data.begin(), h_data.end());
+    std::vector<char> h_mask(mask_size());
+    thrust::sequence(h_mask.begin(), h_mask.end());
```

Comment:
I like to prefer `std` over `thrust` when it does the same thing.

```suggestion
    std::vector<char> h_data(data_size());
    std::iota(h_data.begin(), h_data.end());
    std::vector<char> h_mask(mask_size());
    std::iota(h_mask.begin(), h_mask.end());
```